lib/http/request.js

```diff
@@ -14,7 +14,8 @@ const __defaultSettings__ = {
   use_ssl: false,
   proxy: null,
   timeout: 60000,
-  retry_attempts: 0
+  retry_attempts: 0,
+  default_path: ''
```

Comment:
this change doesn't look related.

Reply:
Removed this, but requests fail if default_path is not set. Yes it is unrelated, because the issue occurs in other situations also, and there is a separate issue for this https://github.com/nightwatchjs/nightwatch/issues/3304. But that needs to be merged before analytics can be shipped.

Reply:
Not reproducible anymore, probably has been fixed.